src/vdev_disk_aio.c

```diff
@@ -217,21 +172,10 @@ vdev_disk_aio_poller(void *arg)
 	while (!vda->vda_stop_polling) {
 		timeout.tv_sec = 0;
 		timeout.tv_nsec = POLL_SLEEP;
-		nr = 0;
 
-		/* First we try non-blocking userspace poll which is fast */
-		if (((struct aio_ring *)(vda->vda_io_ctx))->magic ==
-		    AIO_RING_MAGIC) {
-			nr = user_io_getevents(vda->vda_io_ctx, events);
-		}
-		if (nr <= 0) {
-			/* Do blocking kernel poll */
-			nr = io_getevents(vda->vda_io_ctx, 1,
-			    zfs_vdev_max_active, events, &timeout);
-			VDA_STAT_BUMP(vda_stat_kernel_polls);
-		} else {
-			VDA_STAT_BUMP(vda_stat_userspace_polls);
-		}
+		nr = io_getevents(vda->vda_io_ctx, 1,
+		    zfs_vdev_max_active, events, &timeout);
```

Comment:
should we consider increasing 'timeout'?
I'm also yet to be sure to be the case. Need to look at code. Share your thoughts if you already have.